Primtall -- WIR

martstr opened the pull request.

tjesi commented on src/python/primtall/README.md:
ville heller skrevet "Dobbeltime"

tjesi commented on src/python/primtall/README.md:
Ungdomsskolen også? (gitt at de har en del erfaring i Python, som du nevner senere)

tjesi commented on src/python/primtall/README.md:
Matematikk? (primtall)

tjesi commented on src/python/primtall/README.md:
savner kompetansemål fra samfunssfag, som du har nevnt ovenfor. har du noe fra ungdomsskolematematikk også?

tjesi commented on src/python/primtall/README.md:
"programkode"

tjesi commented on src/python/primtall/README.md:
skal det være en "_" fremfor "vi"?

tjesi commented on src/python/primtall/primtall.md:
Husk TODO

tjesi commented on src/python/primtall/primtall.md:
merk at vi har endrer litt struktur i mappene nå, da tags legges i data.yml istedenfor i oppgaven. sjekk repoet for eksempler.

tjesi commented on src/python/primtall/primtall.md:
n = 2 vil nå gi False

tjesi commented on src/python/primtall/primtall.md:
n = 2 gir False her også

tjesi commented on src/python/primtall/primtall.md:
Liker denne veldig godt!

tjesi commented on src/python/primtall/primtall.md:
sjekke at input er positivt heltall og ikke like 0 eller 1 osv? kan jo også være verdt en kommentar angående sikker programmering at man alltid må verifisere at input er slik det burde være. (dette er jo også grunn til at flere kryptografiske protokoller har blitt brutt opp gjennom årene)

tjesi reviewed (changes requested):
både veiledning og oppgave ser veldig bra ut. har lagt inn noen få kommentarer/spørmål - alt er ikke nødvendigvis noe som må fikses

martstr commented on src/python/primtall/README.md:
Den er grei. Skal alle de fem regnes som dobbeltimer? (Jeg har ingen intuisjon på hvor lang tid et slikt opplegg faktisk tar ...)

martstr commented on src/python/primtall/README.md:
Hvis du mener det, så kan jeg godt senke det ned dit også. Kommer til å prøve å vurdere de andre litt etter hva som er mulig å ta på ungdomsskolen også, men det bør vel erfaringen etter hvert vise ...

martstr commented on src/python/primtall/README.md:
Matematikk og programmering. Samfunnsfag skulle ikke vært der -- copy-paste.

martstr commented on src/python/primtall/README.md:
Jau ...

martstr commented on src/python/primtall/README.md:
Godt sett.

martstr commented on src/python/primtall/README.md:
Copy-paste fra en annen lærerveiledning. Så, opp til deg ...

martstr commented on src/python/primtall/primtall.md:
Ja. Finnes det spoilerbokser som kan kollapses? Eventuelt putter jeg det i en tip-boks.

martstr commented on src/python/primtall/primtall.md:
I repoet syns jeg at jeg så mye dobbelt av dette. Bare for å klargjøre: Det skal *ut* av .md-fila, sant?

martstr commented on src/python/primtall/primtall.md:
Godt sett. Will fix.

martstr commented on src/python/primtall/primtall.md:
Som over.

martstr commented on src/python/primtall/primtall.md:
Godt poeng. Jeg greier ikke helt å bestemme meg her: Vi vil jo gjerne lære bort god programmeringsskikk, så en test av input burde absolutt være med. Samtidig kan jo et slikt tillegg med dets forklaring ta fokus bort fra de andre tingene vi vil ha fram, og dermed gjøre terskelen litt større. Hva tror du?

(Om jeg legger det inn, blir det if not (isinstance(n, (int, long)) and n > 1): <break> return False. Det er litt teknisk, men mindre teknisk enn å sende en exception, som er det man burde gjøre ...)

tjesi commented on src/python/primtall/README.md:
Ja.

tjesi commented on src/python/primtall/README.md:
Ja, gjerne - fint om du inkluderer ungdomsskole. Håper at de skal kunne benyttes av elever under VGS, og så får vi heller justere opp senere dersom det viser seg å ikke fungere.

tjesi commented on src/python/primtall/primtall.md:
ja, vi endret nylig. men noe står dobbelt enda pga de gamle sidene trenger den infoen. vil endres ved bruk av et script ved senere annledning. bare sjekk hvordan vi har gjort det i de andre filene når før du sender PR.

tjesi commented on src/python/primtall/primtall.md:
synes if not (isinstance(n, (int, long)) and n > 1): return False er en god løsning her, som ikke blir for avansert eller tar for mye fokus bort fra hovedtemaet i oppgaven. legg gjerne til dette + en komentar om hva du gjør og hvorfor.

martstr committed:
Merge branch 'master' into primtall

Oisov committed:
Added ALT attributes to every Python exercise. (#674)

* Added ALT attributes to every Python exercise.

* Update hangman2.md

* Update regn_med_lokker.md

* Update skilpaddetekst.md

* Update stjerner_og_galakser.md

Commit:
Add isExternal to data.yml of external courses (#685)

Oisov committed:
Microbit - Hello World! - nb (#676)

* First draft hello world NB

* fixed spelling mistakes, add YAML tags

* Fixed more spelling mistakes

Oisov committed:
Microbit - Buttons - nb (#678)

* Startet å skrive LV for micro:bit buttons

* first draft micro:bit Norwegian

* Improved the YAML header, added liscence, author

* added YAML tags

* Fixed spelling mistakes and sentence structure

Oisov committed:
ALT attributes - web (#680)

* Added ALT attributes to every web exercise

* Update hvor_er_html.md

* Update hei_js.md

Oisov committed:
Microbit - gestures - nb (#681)

* created YAML tags

* Translated gestures exercise to norwegian

Oisov committed:
Style guide - App inventor (#682)

* Style guide from CONVENTIONS.md applied to app inventor exercises

* Fixed errors in 80 character break

Oisov committed:
Microbit - images - nb (#684)

* Translated images exercise to Norwegian

* created YAML tags

Oisov committed:
ALT attributes - Computer Craft (#679)

* Startet å skrive LV for micro:bit buttons

* first draft micro:bit Norwegian

* Improved the YAML header, added liscence, author

* Added ALT tags to computercraft exercises

* Revert "Improved the YAML header, added liscence, author"

This reverts commit 6866d0d096d055785db94d062fbe7c2ff1ceb413.

* Revert "first draft micro:bit Norwegian"

This reverts commit 622f90cdd5958abf5c9a236841fc402859b59c4d.

* Removed file python_buttons_nb

Oisov committed:
ALT attributes - Appinventor (#694)

* Updated ALT attributes for app inventor

* Added conventions formating

Oisov committed:
Added two new formating rules (#696)

Oisov committed:
Fixed formating according to CONVENTIONS and added ALT tags (#695)

Oisov committed:
Microbit - random - nb (#692)

* Started translating microbit_random to nb

* Updated and renamed microbit_random file

* Added YAML tags

* fixed 80 character line break

* Fixed spelling mistakes

Oisov committed:
Microbit - direction - nb (#693)

* Started translating microbit_random to nb

* Completed translating directions and added image

* Added YAML tags

* fixed minor spelling mistakes

Oisov committed:
Conventions - Scratch (#697)

* Fixed formating, added alt tags - Introduction

* Fixed formating, added alt tags - Beginner

* Fixed formating, added alt tags - Experienced

* Fixed formating, added alt tags - Expert

Oisov committed:
Conventions - Python (#698)

* Fixed formating, added alt tags - Introduction

* Fixed formating, added alt tags - Beginner

* Fixed formating, added alt tags - Experienced

* Fixed formating, added alt tags - Expert

Oisov committed:
Microbit - Movement - nb (#699)

* Completed writing python movement

* Added YAML tags

* Fixed minor spelling mistakes

* Fixed URL && 80 character line width

Commit:
Endre navnet på nynorsk fra "Norsk nynorsk" til "Nynorsk" (#701)

* Endre navnet på nynorsk fra "Norsk nynorsk" til "Nynorsk"

* Endre navnet på bokmål fra "Norsk bokmål" til "Bokmål"

mortenaa committed:
Forbedringer etter code review

Commit:
Flytt svenske og danske kurs-info-maler (ikke oversatt ennå) til unpublished. (#703)

mortenaa committed:
Merge branch 'master' into skrive-om-til-ellie

Commit:
data.yml --> course.yml & lesson.yml (rename and create) (#718)

* Rename src/*/data.yml to src/*/course.yml and src/*/*/data.yml to src/*/*/lesson.yml

* Add course.yml to all internal courses, with "isExternal: false"

teodorlu committed:
Merge branch 'master' into skrive-om-til-ellie

teodorlu committed:
Merge pull request #654 from mortenaa/skrive-om-til-ellie

WIP Skriv om elm/02_funksjoner til å bruke Ellie

teodorlu committed:
Fiks restser fra mergekonflikt i Elm-oppgave

Oisov committed:
Conventions && ALT attributes - web (#720)

* Formated web README

* Fixed formating & alt tags - web - introduction

* Fixed formating & alt tags - web - beginner

* Fixed formating & alt tags - web - experienced

* Fixed formating & alt tags - web - expert

* Fixed spacing in lists, removed witespace

* Fixed forgotten titles

Oisov committed:
Conventions && ALT attributes - processing (#721)

* Formated and added ALT tags to processing - introduction

* Formated and added ALT tags to processing - beginner

* Formated every README.md

* Added README, fixed titles

* --append

* Fixed alt tags

* Removed trailing whitespace, fixed spacing titles and lists

Oisov committed:
Conventions && ALT attributes - microbit (#722)

* Fixed ALT tags PXT && formating

* Removed trailing whitespace, fixed spacing titles and lists

Oisov committed:
Conventions && ALT attributes - javafx (#724)

* Added ALT tags, fixed formatting

* Removed trailing whitespace, fixed spacing titles and lists

Oisov committed:
Conventions && ALT attributes - learn2mod (#726)

* Fixed formating and ALT tags

* Added forgotten README

Oisov committed:
Conventions && ALT attributes - arduino (#725)

* Fixed formating and ALT tags

* Removed trailing whitespace, fixed spacing titles and lists

Oisov committed:
Conventions && ALT attributes - misc (#727)

* Fixed misc ALT tags

* Fixed spacing titles, whitespace computercraft

* Fixed forgotten newline unity

* Removed trailing whitespace

Oisov committed:
Conventions && ALT attributes - scratch (README + intro) (#723)

* Fixed ALT inn README and Introductions

* Removed trailing whitespace, fixed spacing titles and lists

steinoro committed:
Added alt-tags for nn

Oisov committed:
Merge branch 'master' into alt-tags-nn

Oisov committed:
Merge pull request #728 from steinoro/alt-tags-nn

Added alt-tags for nn

Oisov committed:
Formated nn, according to conventions

Oisov committed:
Added forgotten files...

steinoro committed:
WIP - Fjerna overflødig tekst (#717)

Sjå i samanheng med arbeidet som pågår med å bygge ein wiki.

Oisov committed:
Conventions && ALT attributes - appinventor (#731)

* Started writing on a syntax_check

* Added a file to check for spacing as well

* Added blank line at the end of files

* Fixed spacing in lists, removed witespace

* Fixed two forgotten files app

* Removed spacing check from branch

* Removed syntax check from branch

Oisov committed:
Conventions && ALT attributes - Python (#729)

* Removed trailing whitespace, spacing titles and lists

* Fixed forgotten titles in README

Oisov committed:
Formated elm, added alt tags (#732)

steinoro committed:
Deleted pages included in wiki (#733)

Oisov committed:
Removed unnececary temp file

Oisov committed:
Merge branch 'master' into conventions_scratch_nn

Oisov committed:
Merge pull request #730 from Oisov/conventions_scratch_nn

Conventions && ALT attributes - Scratch - Nynorsk

Oisov committed:
Fixed formatting on misc files (#742)

Oisov committed:
Added authors to missing files (#746)

* Added Authors to Scratch

* Added Authors to Web

* Added Authors to Python

* Added Authors to Processing

* Added Authors to Arduino

* Added author to appinventor

Oisov committed:
Added authors to missing files V2 (#747)

* Added Authors to Scratch

* Added Authors to Web

* Added Authors to Python

* Added Authors to Processing

* Added Authors to Arduino

* Added author to appinventor

* Fixed double authors

steinoro committed:
Add English version of readme (#748)

Oisov committed:
LKK linter - for markdown and yml files (#752)

* Started writing on a linter for md and lesson.yml files

* Updated path to SRC

Oisov committed:
Added underscore in name

Oisov committed:
Merge pull request #755 from Oisov/rotasjon_scratch_navn.md

Added underscore in name

Oisov committed:
Wrote basic linter

Oisov committed:
fixed links to rotasjon_rundt_punkt (#757)

Oisov committed:
Fixed remaining long lines + microbit yml (#753)

* Fixed forgotten long lines

* Changed footer to translator

* Added forgotten files in scratch

Oisov committed:
Added python tmp files to gitignore (#754)

Oisov committed:
Fixed problems with auto-updating

Oisov committed:
Python script to avoid files being linted twice

Commit:
Merge branch 'master' into LKK_auto_linter

Oisov committed:
Merge pull request #756 from Oisov/LKK_auto_linter

Wrote basic linter

Skagevang committed:
Remove playlist for kodu and fix yml for fargespill (#763)

* remove playlist for kodu and fix yml for fargespill

* the link was broken

Oisov committed:
Fixed minor bugs with code blocks

Oisov committed:
Merge branch 'master' into LKK_linter_auto

Oisov committed:
Merge branch 'master' of github.com:kodeklubben/oppgaver into LKK_linter_auto

Oisov committed:
Removed backtick linting

Oisov committed:
Merge branch 'LKK_linter_auto' of github.com:Oisov/oppgaver into LKK_linter_auto

Oisov committed:
commented check to fix backticks

steinoro committed:
Omsetting av diverse tekstar Scratch (#764)

Skagevang committed:
removed logo (#766)

Commit:
Merge branch 'master' into LKK_linter_auto

Oisov committed:
Merge pull request #765 from Oisov/LKK_linter_auto

[linter] Fixed minor bugs with code blocks

Oisov committed:
added comments, and check for . in titles

Oisov committed:
Merge pull request #772 from Oisov/LKK_linter_update

[linter] added comments, and check for . in titles

Commit:
Remove some misplaced characters

Oisov committed:
Merge branch 'master' into fix_typo_in_yml

Oisov committed:
Merge pull request #785 from NorwegianKiwi/fix_typo_in_yml

Remove some misplaced characters

Oisov committed:
Auto linted every lesson.yml file

Oisov committed:
Merge pull request #789 from Oisov/lesson_yml_linter

Auto linted every lesson.yml file

steinoro committed:
CSS: skjul ninjaene på nynorsk (#792)

steinoro committed:
CSS: layout på nynorsk (#788)

steinoro committed:
CSS: animasjon på nynorsk (#786)

steinoro committed:
Skilpaddeskolen på nynorsk (#784)

steinoro committed:
Skilpadder hele veien på nynorsk (#783)

steinoro committed:
Tre på rad på nynorsk (#780)

steinoro committed:
Tre på rad 2 på nynorsk (#782)

steinoro committed:
Repetisjon på nynorsk (#781)

steinoro committed:
Tekst ABC på nynorsk (#779)

steinoro committed:
Skilpadder på nynorsk (#777)

steinoro committed:
Enkle objekter på nynorsk (#770)

steinoro committed:
Sprettball på nynorsk (#778)

steinoro committed:
Lister og indekser på nynorsk (#774)

steinoro committed:
Fargespill på nynorsk (#769)

* Fargespill på nynorsk

* Update fargespill_nn.md

Commit:
fjerne url fra fargespill (#793)

siden brekker pga ustabil nettside

steinoro committed:
Bokstaver på nynorsk (#771)

steinoro committed:
Hangman på nynorsk (#768)

steinoro committed:
Gjettelek på nynorsk (#767)

steinoro committed:
CSS: skjule ninja - fix alt texts  (#794)

* CSS: skjul ninjaene på nynorsk

* Fix alt texts

steinoro committed:
Løpende strekmann på nynorsk (#776)

* Løpende strekmann på nynorsk

* Fix invisible links

steinoro committed:
Hemmelige koder på nynorsk (#773)

* Hemmelige koder på nynorsk

* Add missing letters

steinoro committed:
Ordbøker på nynorsk (#775)

* Ordbøker på nynorsk

* Fix checkbox

steinoro committed:
HTML: Introduksjon til Web på nynorsk (#797)

steinoro committed:
HTML: forsvunnet katt på nynorsk (#796)

* Fix line length

* HTML: forsvunnet katt på nynorsk

steinoro committed:
HTML: del inn nettsiden på nynorsk (#795)

* Fix link and code block

* HTML: Del inn nettside på nynorsk

steinoro committed:
HTML: lenk sammen nettsider på nynorsk (#798)

steinoro committed:
CSS: style nettsider på nynorsk (#791)

* CSS: style nettsider på nynorsk

* Fix alt texts

steinoro committed:
CSS: tekststil på nynorsk (#790)

* CSS: tekststil på nynorsk

* Update tekststil_nn.md

Skagevang committed:
Fikset license i yml (#799)

* fixed license

* quotefail...

* small fix for instruction lesson

* added back license for index

steinoro committed:
HTML: din egen nettside på nynorsk (#800)

steinoro committed:
HTML: lyd og video på nynorsk (#802)

* HTML: lyd og video på nynorsk

* Fjerna lisens

Oisov committed:
Microbit - network - nb (#716)

* Added YAML tags

* Added images

* Translated microbit network to nb_NO

* Fixed spelling mistakes

* Renamed data.yml to lesson.yml

* fjernet lisens fra oppgave

* la til lisens i lesson.yml

Oisov committed:
Microbit - Input output - nb (#713)

* Added YAML header

* Wrote inputt outputt in microbit

* Added images

* Renamed data.yml to lesson.yml

* Fixed spelling, added info about GND

steinoro committed:
HTML: publiser på nynorsk (#803)

* HTML: publiser på nynorsk

* Oppdater bilete

* Slett feil biletefil

* Legg til bilete

steinoro committed:
HTML: bygge hjemmeside på nynorsk (#801)

* HTML: bygge hjemmeside på nynorsk

* Fjerna lisens

steinoro committed:
JS: Hei, JavaScript på nynorsk (#813)

* Fiksa kodeblokk

* JS: Hei, JavaScript på nynorsk

steinoro committed:
JS: Grunnleggende JS på nynorsk (#812)

steinoro committed:
JS: Partikkel-animasjon på nynorsk (#814)

steinoro committed:
Scratch: Breakout på nynorsk (#819)

* Breakout på nynorsk

* Kodeblokker på nynorsk

* Kodeblokker på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
Kuprat på nynorsk (#825)

steinoro committed:
If-setninger på nynorsk (#824)

steinoro committed:
Kryptonøtt på nynorsk (#823)

steinoro committed:
NNkalkulator (#822)

steinoro committed:
Hvor gammel er du på nynorsk (#821)

steinoro committed:
HTML: hvor er HTML på nynorsk (#818)

steinoro committed:
Huskespill på nynorsk (#820)

steinoro committed:
JS: partikkelfest på nynorsk (#817)

steinoro committed:
JS: partikkel-gravitasjon på nynorsk (#816)

steinoro committed:
Fiks kodeblokker (#829)

steinoro committed:
Scratch: Breakout - fiks kodeblokk (#830)

* Fiks kodeblokker

* Fiks kodeblokk

steinoro committed:
Scratch: 3D-flakser del 1 på nynorsk (#826)

* Kodeblokker på nynorsk

* Kodeblokker på nynorsk - fiks

* Fiksa kodeblokker

steinoro committed:
Scratch: Felix og Herbert på nynorsk (#831)

* Kodeblokker på nynorsk

* Fiksa kodeblokk

steinoro committed:
Scratch: Astrokatt på nynorsk (#828)

* Kodeblokker på nynorsk

* Fiksa kodeblokker

Oisov committed:
Started working on version .2 of the linter

steinoro committed:
Scratch: Asteroids på nynorsk (#827)

* 3D-flakser 2 på nynorsk

* Kodeblokker på nynorsk

* Kodeblokker på nynorsk

* Slett fil som ikkje høyrer til denne PR

* Fiksa kodeblokker

* Endre -> endra

steinoro committed:
Scratch: Bursdag i Antarktis på nynorsk (#810)

* Bursdag i Antarktis på nynorsk

* Kodeblokker på nynorsk

* Fiks kodeblokker

* Fiksa kodeblokk

Oisov committed:
LEGO Mindstorm - nb (#700)

* LEGO Mindstorms - introduction - translated

* LEGO Mindstorms - beginner - translated

* LEGO Mindstorms - experienced - translated

* LEGO Mindstorms - expert - translated

* Removed external files

* Replaced '# Oppgave [1-9].[1-9] - title' with '# title'

* Updated playlists

* Removed unnececary image

steinoro committed:
Scratch: Bruke data på nynorsk (#807)

* Bruke data på nynorsk

* Kodeblokker på nynorsk

* Fiksa kodeblokk

steinoro committed:
Endra tittel til å følge rettskrivingsreglar (#833)

steinoro committed:
Scratch: Donkey Kong på nynorsk (#808)

* Donkey Kong på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
Scratch: 3D-flakser 2 på nynorsk (#805)

* 3D-flakser 2 på nynorsk

* Kodeblokker på nynorsk

* Fiks kodeblokker

* Endre -> endra

* Riktig språkkode

steinoro committed:
Kart på nynorsk (#735)

steinoro committed:
Scratch: Enarmet banditt på nynorsk (#706)

* Enarmet banditt på nynorsk

* Fjerna lisens

* Kodeblokker på nynorsk

* Fiks kodeblokker

steinoro committed:
Scratch: Flaksefugl på nynorsk (#804)

* Flaksefugl på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
JS: trykkomania på nynorsk (#815)

* Fiks filnamn

* JS: trykkomania på nynorsk

* Retta filnamn i README

steinoro committed:
Scratch: Flagg på nynorsk (#809)

* Flagg på nynorsk

* Kodeblokker på nynorsk

* Fiksa kodeblokker

steinoro committed:
Scratch: Fyrverkeri på nynorsk (#806)

* Fyrverkeri på nynorsk

* Kodeblokker på nynorsk

* Fiksa kodeblokker

steinoro committed:
Kodeblokker på nynorsk (#834)

steinoro committed:
Scratch: Lag spill på nynorsk (#739)

* Lag spill på nynorsk

* Fjerna lisens

* Fjerna logo og lisens

steinoro committed:
Scratch: Krabbeangrep på nynorsk (#737)

* Krabbeangrep på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
Scratch: Julekort på nynorsk (#736)

* Julekort på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
Scratch: King Kong på nynorsk (#734)

* King Kong på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
Scratch: Jafsefisk på nynorsk (#719)

* Jafsefisk på nynorsk

* Fjerna lisens

* Kodeblokker på nynorsk

steinoro committed:
Scratch: Hvor i all verden 3 på nynorsk (#714)

* Hvor i all verden 3 på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
Scratch: Hvor i all verden 1 på nynorsk (#712)

* Hvor i all verden 1 på nynorsk

* Kodeblokker på nynorsk

* Kodeblokker på nynorsk

* Fiksa kodeblokk bokmål

steinoro committed:
Scratch: Hva er det på nynorsk (#709)

* Hva er det på nynorsk

* Fjerna lisens

* Kodeblokker på nynorsk

steinoro committed:
Scratch: Halloweenanimasjon på nynorsk (#707)

* Halloweenanimasjon på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
Python: tekstfiler (#842)

* Index på nynorsk

* Oppdatert index på bokmål

* Index på engelsk

steinoro committed:
PGZ Hangman på nynorsk (#841)

steinoro committed:
Python: Stjerner og galakser på nynorsk (#840)

steinoro committed:
Python: Skilpaddetekst på nynorsk (#839)

steinoro committed:
Python: Skilpaddefraktaler på nynorsk (#837)

steinoro committed:
Python: Regn med løkker på nynorsk (#835)

steinoro committed:
Processing: Tegnespillet på nynorsk (#846)

steinoro committed:
Processing: Farger på nynorsk (#843)

steinoro committed:
Python: Mattespill på nynorsk (#836)

steinoro committed:
Scratch: Pong på nynorsk (#744)

* Pong på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
Scratch: Rotasjon rundt egen akse på nynorsk (#760)

* Fjerna læringsmål

Læringsmåla høyrer heime og er inkludert i lærarrettleiinga (readme)

* Rotasjon rundt akse på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
Scratch: Gangemesteren på nynorsk (#832)

* Kodeblokker på nynorsk

Omsett kodeblokker. Endra filnamn til å følgje standard.

* Bilete på nynorsk

* Fiksa kodeblokk

steinoro committed:
Scratch: Straffespark på nynorsk (#759)

* Straffespark på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
Scratch: Soloball på nynorsk (#811)

* Soloball på nynorsk

* Kodeblokker på nynorsk

steinoro committed:
Scratch: Tegneprogram på nynorsk (#762)

* Tegneprogram på nynorsk

* Fjerna lisens

* Kodeblokker på nynorsk

steinoro committed:
Scratch: Rotasjon rundt punkt på nynorsk (#761)

* Rotasjon rundt et punkt på nynorsk

* Fjerna læringsmål frå oppgåva

* Kodeblokker på nynorsk

Skagevang committed:
fix two bugs (#847)

Commit:
Merge branch 'master' into primtall

Skagevang committed:
file ending fix (#851)

steinoro committed:
Scratch: Hoppehelt på nynorsk (#708)

* Hoppehelt på nynorsk

* Kodeblokker på nynorsk

* Fiksa kodeblokker

steinoro committed:
Scratch: Spøkelsesjakten på nynorsk (#758)

* Spøkelsesjakten på nynorsk

* Fjerna lisens

* Kodeblokker på nynorsk

* Fiksa lenke til bilete

steinoro committed:
Scratch: Snurrige figurar på nynorsk (#751)

* Snurrige figurar på nynorsk

* Kodeblokker på nynorsk

* Fiksa lenke til bilete

steinoro committed:
Scratch: Snake på nynorsk (#749)

* Snake på nynorsk

* Kodeblokker på nynorsk

* Fiksa lenke til bilete

steinoro committed:
Scratch: Norgestur på nynorsk (#740)

* Norgestur på nynorsk

* Kodeblokker på nynorsk

* Fiksa kodeblokker

steinoro committed:
Scratch: Redd verden på nynorsk (#745)

* Rettskriving helikopter

* Redd verden på nynorsk

* Kodeblokker på nynorsk

* Fiksa kodeblokk

steinoro committed:
Scratch: Pingviner på tur på nynorsk (#743)

* Pingviner på tur på nynorsk

* Kodeblokker på nynorsk

* Fiksa kodekloss

steinoro committed:
Scratch: Ørkenløp på nynorsk (#741)

* Ørkenløp på nynorsk

* Fjerna lisens

* Kodeblokker på nynorsk

* Fiksa kodeblokker

steinoro committed:
Scratch: Labyrint på nynorsk (#738)

* Labyrint på nynorsk

* Kodeblokker på nynorsk

* Fiksa kodeblokk

steinoro committed:
Scratch: Hvor i all verden 2 på nynorsk (#715)

* Hvor i all verden 2 på nynorsk

* Kodeblokker på nynorsk

* Lasta opp bilete på nynorsk

steinoro committed:
Processing: oppdatert index-filer (#850)

* Oppdatert tekst på bokmål

* Fullført tekst på engelsk

* Omsett tekst til nynorsk

steinoro committed:
Web: Del inn nettsider - fiks lenker (#848)

* Oppdatert lenker

Frå http til https

* colorpicker.com støtter ikke https

steinoro committed:
Processing: Sprettende ball på nynorsk (#849)

steinoro committed:
Processing: Tetris på nynorsk (#852)

steinoro committed:
Processing: Mangekanter på nynorsk (#845)

steinoro committed:
Processing: Ping pong på nynorsk (#844)

steinoro committed:
Python: Skilpaddekunst på nynorsk (#838)

* Python: Skilpaddekunst på nynorsk

* Fiksa lenke til bilete

Oisov committed:
Replaced old linter dir with new linter

steinoro committed:
Processing: Verden del 1 på nynorsk (#856)

steinoro committed:
Processing: Trigonometri på nynorsk (#855)

steinoro committed:
Processing: To player på nynorsk (#854)

* Processing: To player på nynorsk

* Skrivefeil

steinoro committed:
Scratch: Snøballkrig på nynorsk (#750)

* Snøballkrig på nynorsk

* Kodeblokker på nynorsk

* Endra skrivemåte for helt og skumling

Oisov committed:
Added improved documentation for code

Oisov committed:
Merge branch 'master' into LKK_linter_2

Oisov committed:
Merge pull request #858 from Oisov/LKK_linter_2

LKK linter V.2

Skagevang committed:
CodeStudio på nynorsk og engelsk (#864)

* moved kodu to unpublished and added nn and en to codestudio lessons

* changed title for 4 english lessons

* changed titles

steinoro committed:
Processing: Verden del 2 på nynorsk (#857)

* Processing: Verden del 2 på nynorsk

* Fiksa linjelengde

steinoro committed:
Microbit: Hermegåsa på nynorsk (#862)

* Microbit: Hermegåsa på nynorsk

* skriveleif

steinoro committed:
Microbit: Terning på nynorsk (#863)

* Microbit: Terning på nynorsk

* change language

nb to nn

steinoro committed:
Microbit: Stein, saks, papir på nynorsk (#861)

* Microbit: Stein, saks, papir på nynorsk

* Fiksa lenker til bilete

steinoro committed:
Microbit: Frustrasjon på nynorsk (#860)

* Fiksa lisens og kreditering

* Microbit: Frustrasjon på nynorsk

steinoro committed:
Microbit: Gangespill på nynorsk (#859)

Oisov committed:
Added primitive textwraper, removed yml_temp

Oisov committed:
Added primitive textwraper, removed yml_temp

Oisov committed:
Merge branch 'LLK_linter_2' of github.com:Oisov/oppgaver into LLK_linter_2

Oisov committed:
Remove python files used for testing

Oisov committed:
Remove python files used for testing

Oisov committed:
Merge branch 'LLK_linter_2' of github.com:Oisov/oppgaver into LLK_linter_2

Oisov committed:
Fixes missing newlines, missing whitespace in lesson.yml

Oisov committed:
Fixed bug with titles being matched in codeblocks

Oisov committed:
Fixed bug with titles being matched in codeblocks

Oisov committed:
Fixed merge conflicts

Oisov committed:
Merge pull request #865 from Oisov/LLK_linter_2

Linter - added textwraper

Oisov committed:
Fixes for autowrapper again...

Oisov committed:
Merge pull request #866 from Oisov/LKK_linter_3

Linter - Fixes for autowrapper again...

Oisov committed:
Sorted keys and tags in lesson.yml

Oisov committed:
Merge pull request #867 from Oisov/lesson_yml_sort

Sorted keys and tags in lesson.yml

Oisov committed:
First working version of new linter

Oisov committed:
Moved new linter over to auto_md.py

Oisov committed:
Merge branch 'master' into linter_v4

Oisov committed:
Merge pull request #868 from Oisov/linter_v4

Linter v4 - Fix for codeblocks & html

Oisov committed:
Misc bugfixes

Oisov committed:
Removed printing

Oisov committed:
Merge pull request #869 from Oisov/linter_v4.1

Linter v4.1 - Misc bugfixes

Oisov committed:
Fixed backtick bug

Oisov committed:
Added check to check if all codeblocks are closed

Oisov committed:
Merge pull request #870 from Oisov/linter_v4.2

Linter v4.2 - More fixes to codeblock formating

Oisov committed:
Added check to match html blocks

Oisov committed:
Merge pull request #872 from Oisov/linter_v4.3

Linter - Added check to match html blocks

Commit:
Legg til lesson.yml med indexed=false til veiledninger, og del opp slik at det bare er én veiledning per folder. Fiks alle referanser. (#881)

kinegei committed:
Processing - nytt spill: bildepresentasjon (#531)

* Processing - nytt spill: bildepresentasjon

* Oppdatert i henhald til kommentarar, formatering

kinegei committed:
Processing - nytt spill: rulling av bilder (#532)

* Processing - nytt spill: rulling av bilder

* Processing: oppdatering på rulling av bilder, nytt navn og fiksa feil

* Oppdatert i henhald til kommentarar, formatering

* Lagt til biletetekstar

steinoro committed:
Fiksa filnamn, fjerna unøvendige filer (#884)

steinoro committed:
Processing: Uendelig bakke - LV (#885)

steinoro committed:
Processing: Bildepresentasjon - LV (#886)

steinoro committed:
Microbit: Buttons på nynorsk (#887)

Commit:
Merge branch 'master' into primtall

Commit:
Update lesson.yml (#888)

Commit:
Add lesson files in processing (#890)

* lesson.yml for bildepresentasjon

- added lesson.yml
- sorted header in readme

* lesson.yml for uendelig_rulling

- added lesson.yml
- sorted header in readme

gahjelle committed:
Lagre leaflet.js lokalt (#883)

* Oppdater til siste versjon av Kartverkets topo-kart

* Last ned Leaflet (som brukes til å vise kart) lokalt. Versjon 1.3.3

martstr committed:
lesson.yml

martstr committed:
Reviewer comments

Commit:
fix 404 error in computercraft (#891)

removed an / in the end of url

Commit:
Merge branch 'master' into primtall
